common/build.gradle

```diff
@@ -41,7 +41,6 @@ dependencies {
     // These will be dependencies not packaged with the .jar
     // They need to be provided either through the database or in an extra .jar
     compileOnly group: 'org.neo4j', name: 'neo4j', version: neo4jVersionEffective
-    compileOnly group: 'org.apache.commons', name: 'commons-configuration2', version: '2.9.0'
```

Comment:
The Snyk build failure is because of two newly found vulnerabilities in commons-configuration2. I will mitigate by this change plus a bump of the version in Neo4j